his/core/application-integration-3270designer-2.md

```diff
@@ -0,0 +1,64 @@
+---
+title: What is the 3270 Design Tool
+description: Get started with the 3270 Design tool for Logic Apps, development tool available for creating green screen applications
+author: haroldcampos
+ms.author: "hcampos"
+ms.prod: "host-integration-server"
+ms.topic: overview 
+ms.date: "10/17/2023"
+
+#CustomerIntent: As a programmer I need to use the 3270 design tool to create metadata for the Logic Apps connector for 3270
+---
+
+# What is the 3270 Design Tool? 
+
+Learn how to create Mainframe metadata 3270 stream based artifacts for Azure Logic Apps.
+
+## What is the 3270 Design Tool
+
+The 3270 Design Tool is a desktop application used to define metadata used by the IBM 3270 Connector for Azure Logic Apps to navigate and interact with your 3270 applications.
+
+:::image type="content" source="media/la-3dtdesigner-main-view1.png" alt-text="HIS Designer for Logic Apps":::
+
+## Why do we need a Tool for interacting with 3270 Applications?
```

Comment:
Avoid ["we"](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we):

```suggestion
## Why use this tool to interact with 3270 applications?
```